package.json

```diff
@@ -21,14 +21,14 @@
   },
   "dependencies": {
     "ajv": "6.3.0",
-    "async": "2.6.0",
+    "gzip-js": "0.3.2",
     "jszip": "3.1.5",
-    "gzip-js": "0.3.2"
+    "pify": "^4.0.1"
```

Comment:
I think we should pin this dependency? I think usually we pin "dependencies" but allow in range updates for "devDependencies"...